packages/sanity/src/core/form/studio/FormBuilder.tsx

```diff
@@ -175,6 +136,50 @@ function RootInput() {
     [onSetFieldSetCollapsed],
   )
 
+  const {file, image} = useSource().form
```

Comment:
This can be removed since it is not used anywhere!

Reply:
Nice catch! Removed